### ISSUE TITLE
[Feature Request] Different Coloured inputs in terminus.

### ISSUE THREAD

**Issue by arshapplied** [randy3k/Terminus#287]
Hey there I am using a user theme in sublime terminus , with a plain black background and white outputs.

Is there a way to have inputs show as blue colours, and the errors as red colours. (Something like a Molokai theme).

Thanks.

**Comment by randy3k** [comment]
What you specify in the settings are the actual color choice for the ANSI colors for the terminal. 
The terminal determines which ANSI color is used in each position of the terminal. The color scheme has no controls what is highlighted in what color.

**Comment by arshapplied** [comment]
So is it possible to have different coloured inputs in the terminal ?

**Comment by randy3k** [comment]
Maybe (depends on your shell), but it is irrelevant to Terminus.

**Comment by arshapplied** [comment]
Do you think I can use terminus or your other plugins to run my code, in Mac terminal app. I am using terminus to open the terminal in the bottom of my pane. But I would like it to use the terminal app on my Mac to run the same command.

I tried using "terminus_open", but it still makes use of Sublime to open the terminal. 

All I want it to do, after clicking build :
1. if terminal app is not open , open it , and show the output of the code.
2. if terminal app is open, and is already displaying output of a different file, show the output of the file that gave the latest build command.

Basically what terminus is doing by default, but in the bottom pane.
